WORKSPACE

```diff
@@ -56,6 +56,17 @@ dpkg_list(
         "libpython2.7-minimal",
         "python2.7-minimal",
         "libpython2.7-stdlib",
+
+        #dotnet
+        "libcurl3",
+        "libgcc1",
+        "libstdc++6",        
```

Comment:
libgcc1 and libstdc++6 are dupes with Java.

Reply:
ok